packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.vue

```diff
@@ -79,8 +145,23 @@ const {
       >
         {{ option.label }}
       </OnyxListboxOption>
+
+      <li v-if="loadingMode === 'button' && !isEmpty" class="onyx-listbox__slot">
+        <OnyxButton
+          class="onyx-listbox__loading-button"
+          :label="loadMoreButtonLabel"
+          mode="plain"
+          :icon="plusSmall"
+          :loading="props.loading"
+          @click="emit('loadMore')"
+        />
+      </li>
     </ul>
 
+    <div v-if="props.loading && loadingMode === 'lazy' && !isEmpty" class="onyx-listbox__slot">
```

Comment:
we're already in an v-else block which excludes isEmpty to be truthy, so this is redundant

```suggestion
    <div v-if="props.loading && loadingMode === 'lazy'" class="onyx-listbox__slot">
```